src/choose-location/choose-location.component.tsx

```diff
@@ -26,7 +26,7 @@ export default function ChooseLocation(props: ChooseLocationProps) {
         .catch(createErrorHandler());
     }
     return () => abortController.abort();
-  }, [isSubmitting]);
+  }, [config.links.loginSuccess, isSubmitting, location, props]);
```

Comment:
I'm asking about this in https://twitter.com/brandard_err/status/1223295010122141696 ... I'm not sure we should change these? This calls the handler every time any one of these four things changes, right?

Reply:
Can we make this PR just about fixing the placement, and split the useEffect dependency stuff into another PR? I really wanna approve the placement stuff quick.

Reply:
> Can we make this PR just about fixing the placement, and split the useEffect dependency stuff into another PR? I really wanna approve the placement stuff quick.

Agree will fix the useEffect dependency with another PR.